### PR TITLE
Add `Manage` functionality and unzip feature.

### PR DESCRIPTION
**Pull request by TranquilRock**
View inflations are also changed into view bindings.

### CODE REVIEW

File: app/src/main/java/com/tranquilrock/androidscript/App.kt
Comment (Roamer1337) [inline app/src/main/java/com/tranquilrock/androidscript/App.kt:1]: Crashing the file is not a good idea.

File: app/src/main/java/com/tranquilrock/androidscript/component/editor/BlockViewHolder.kt
Comment (Roamer1337) [inline app/src/main/java/com/tranquilrock/androidscript/component/editor/BlockViewHolder.kt:1]: The font will be invisible in darkmode.

**Review by Roamer1337 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/tranquilrock/androidscript/App.kt
Comment (TranquilRock) [reply app/src/main/java/com/tranquilrock/androidscript/App.kt:1]: Indeed, but I would expect to handle the errors as the next milestone. The app is not crashing in my trial, or I can mark the release as an alpha release.

File: app/src/main/java/com/tranquilrock/androidscript/component/editor/BlockViewHolder.kt
Comment (TranquilRock) [reply app/src/main/java/com/tranquilrock/androidscript/component/editor/BlockViewHolder.kt:1]: Composition and styling are postponed now.
We could issue it as a reminder.

Comment (Roamer1337) [reply app/src/main/java/com/tranquilrock/androidscript/component/editor/BlockViewHolder.kt:1]: SGTM

File: app/src/main/java/com/tranquilrock/androidscript/App.kt
Comment (Roamer1337) [reply app/src/main/java/com/tranquilrock/androidscript/App.kt:1]: Sure, but the previous version `exit process` is used when openCV is not loaded. 
Why do we remove that here?

Comment (TranquilRock) [reply app/src/main/java/com/tranquilrock/androidscript/App.kt:1]: That solution was found on StackOverflow, but I found it to cause a restart loop on emulators.
The root cause is still unclear.

Comment (Roamer1337) [reply app/src/main/java/com/tranquilrock/androidscript/App.kt:1]: I see. We have better resolve this ASAP though.

**Review by Roamer1337 — APPROVED** [review]

**Comment by Roamer1337** [comment]
Great, I'll issue the drak mode issue latter.

**Pull request merged**